app/helpers/time_language_helper.rb

```diff
@@ -0,0 +1,66 @@
+module TimeLanguageHelper
+  def time_diff_in_natural_language(from_time, to_time)
```

Comment:
This is a helper to add `(_ years, _ months, _ days late)`.

I am not sure if these values are translated and am not certain how to test for that.